src/webui/api/torrentscontroller.cpp

```diff
@@ -292,7 +296,11 @@ void TorrentsController::propertiesAction()
 // The dictionary keys are:
 //   - "url": Tracker URL
 //   - "status": Tracker status
-//   - "num_peers": Tracker peer count
+//   - "tier": Tracker tier
+//   - "num_peers": Tracker received count
```

Comment:
Received count of what?

Reply:
Not quite sure, but this is the nomenclature used in the GUI.

Reply:
I investigated further, it's the total number of peers the torrent is connected to. It's documented [here](https://libtorrent.org/include/libtorrent/torrent_status.hpp) for `torrent_status::numPeers`.